app/src/common/shared/org/mozilla/vrbrowser/ui/widgets/dialogs/UIDialog.java

```diff
@@ -39,13 +41,33 @@ public boolean isDialog() {
         return true;
     }
 
+    @Override
+    public void show(int aShowFlags) {
+        super.show(aShowFlags);
+
+        mWidgetManager.pushWorldBrightness(this, WidgetManagerDelegate.DEFAULT_DIM_BRIGHTNESS);
+    }
+
+    @Override
+    public void hide(int aHideFlags) {
+        super.hide(aHideFlags);
+
+        mWidgetManager.popWorldBrightness(this);
+    }
+
     // WidgetManagerDelegate.FocusChangeListener
 
     @Override
     public void onGlobalFocusChanged(View oldFocus, View newFocus) {
-        if (!ViewUtils.isEqualOrChildrenOf(this, newFocus) && isVisible()) {
+        if (oldFocus == this && isVisible()) {
```

Comment:
Is this reliable? what happens if a children view gets the focus?

Reply:
Do we know why `ViewUtils.isEqualOrChilderenOf()` was failing on the first open of the share dialog?

Reply:
If this doesn't cause any known issues I think we should land it and address in a follow up.

Reply:
@bluemarvin was failing because when showing the sendtabs dialog we hide the radiobutton while the devices are loading. That made the activity change the focus to null but only if the activity has been paused as the activity seems to reset the focus to null in that case.

@MortimerGoro This is how it was before https://github.com/MozillaReality/FirefoxReality/commit/d0cd20b151acbe579564b23a7f1fd90f707b90ac#diff-4f3b7fc6e7c2e6cb9d5359555ffe7e67R46 but for what Ive seen it wasn't reliable and most dialogs just used on the world click listener to hide. 

Actually I've just removed the global focus change listener from UIDialog as it's not really necessary.